.github/workflows/lint-examples.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint Example Workflows
```

Comment:
Please update this file to use two spaces for an indent level instead of four to match the other files.